pipeline/nfcore_custom_sarek.config

```diff
@@ -0,0 +1,16 @@
+/*
+ * -------------------------------------------------
+ *  nfcore custom profile Nextflow config file
+ * -------------------------------------------------
+ * Config options for all custom environments.
```

Comment:
It would be nice to update this a little to explain the difference to the main config profiles and mention that it's not necessary for all institutes / pipelines.

Reply:
I'm mainly playing now, but will make docs for that.
so that we can do that for UPPMAX as well

Reply:
Added some docs.
Please tell me what do you think about it.